.pre-commit-config.yaml

```diff
@@ -13,10 +13,12 @@
 # limitations under the License.
 repos:
   - repo: https://github.com/golangci/golangci-lint
-    rev: v1.43.0
+    rev: v1.51.1
     hooks:
       - id: golangci-lint
+        log_file: /tmp/pre-commit.log
   - repo: https://github.com/pre-commit/mirrors-prettier
     rev: v2.4.1
     hooks:
       - id: prettier
+        log_file: /tmp/pre-commit.log
```

Comment:
I would prefer to create the file in the project and add it to the `.gitignore` and `.dockerignore` files, so when working from the IDE you don't have to traverse directories outside of the project